### ISSUE TITLE
Can't listen all filter was unchecked event. How can i listen them?

### ISSUE THREAD

**Issue by eggplantiny** [nhn/tui.grid#1020]
Can't listen all filter was unchecked event. How can i listen them?

I wanna get count of filtered items so get them with using filter event.
but can't get them when all filters are turned off.

![zxc](https://user-images.githubusercontent.com/57978095/80554988-dae4fc00-8a09-11ea-9a6f-cb4bd97eac1b.png)

in this screenshot, i can catch count of items when i check filter.
but, if i reclicked all of filter, can't catch any event.

version.
"@toast-ui/vue-grid": "^4.10.0"
"vue": "^2.6.11"

**Comment by js87zz** [comment]
@eggplantiny 
When there are no filtered items, In other words, all checkboxes are checked, the `filter` event is not triggered. You mean that right?
We will fix it and comment here after fixing it.

**Comment by js87zz** [comment]
@eggplantiny 
The bug has been fixed in `v4.12.1`(#1047).
We created the new event which is triggered after releasing the condition of filtering to reduce confusion.
```js
// triggered after releasing the condition of filtering
grid.on('afterUnfilter', () => {
  // ...
});

// triggered after filtering the data. this event is same as `filter` event.
// However, `filter` event will be deprecated and we recommend to use this event
grid.on('afterFilter', () => {
  // ...
});
```
* related issue(#1042)

**Comment by eggplantiny** [comment]
@js87zz thx for your efforts 💪💪💪
